docs/ntppool/zh_CN/join/configuration.html

```diff
@@ -0,0 +1,81 @@
+<div class="block">
+
+<h3>加入NTP服务器群的推荐系统设定</h3>
+
+<p>
+<a href="http://support.ntp.org/">support.ntp.org</a>上的<a href="http://support.ntp.org/bin/view/Support/WebHome">帮助中心</a>包含了很多有用的信息。
+</p>
+
+<p>
+如果你仅仅是想要<i>使用</i>服务器群，请参阅<a href="/use.html">如何使用服务器群</a>。
+</p>
+
+<p>
+如果你需要有关NTPD软件的帮助，推荐你访问
+<a href="http://groups.google.com/group/comp.protocols.time.ntp">comp.protocols.time.ntp</a>。
+</p>
+
+<p>
+如果你意欲加入时间服务器群，你需要注意以下几点：
+</p>
+
+<h4 id="management-queries">管理查询（Management queries）</h4>
+
+<p>请将默认设置中的管理查询（management queries）设定为阻止。
+如果你使用NTPD，你需要在默认的限制（restrict）列表中添加非查询（noquery）操作，例如：</p>
+
+[% INCLUDE "join/configuration/restrict-default.html" %]
+
+<p>为使诸如"ntpq -c pe"之类的命令在本地（localhost）得以正常工作，你需要进行如下操作：</p>
+
+[% INCLUDE "join/configuration/restrict-localhost.html" %]
+
+<h4>设置5个左右的上游服务器</h4>
+
+<p>
+NTPD至少要连接三个服务器才能正常工作。（只有一块手表，可以知道时间；
+拥有两块或两块以上的手表并不能告诉一个人更准确的时间，反而会让看表的人失去对准确时间的信心。--手表定律）
+</p>
+
+<p>
+对于在群中的服务器，我们推荐你设置4~7台上游服务器。
+</p>
+
+
+<h4>请勿使用任何*.pool.ntp.org的服务器</h4>
+
+<p>
+如果你意欲加入服务器群，为了保证优质的服务，请不要使用任何*.pool.ntp.org的服务器。
+</p>
+
+<p>
+为了保障服务器群的稳定运行，建议你自行挑选一些优质且地理位置靠近的时间服务器。
+NTP.org的维基页面包含了<a href="http://support.ntp.org/bin/view/Servers/WebHome">公共服务器列表</a>。
+</p>
+
+
+<h4>使用统一的NTPD软件</h4>
+
+<p>
+我们尊重软件的多元化，然而很大一部分的故障皆是由于使用非NTPD软件。
+</p>
+
+<p>
+如果你只是想<i>使用</i>服务器群，你可以使用任何通过NTP协议沟通的软件。但如果你想加入服务器群，我们推荐你使用
+<a href="http://support.ntp.org/bin/view/Main/SoftwareDownloads">ntpd</a>。
+</p>
+
+
+<h4>！请勿使用本地时钟驱动！</h4>
```

Comment:
`LOCAL` is not an emphasis; it's just literally "LOCAL". http://doc.ntp.org/4.2.0/drivers/driver1.html